server/fleet/scripts.go

```diff
@@ -285,14 +290,24 @@ func (hsr HostScriptResult) UserMessage(hostTimeout bool) string {
 
 	switch *hsr.ExitCode {
 	case -1:
-		return RunScriptScriptTimeoutErrMsg
+		return HostScriptTimeoutMessage(hostTimeoutValue)
 	case -2:
 		return RunScriptDisabledErrMsg
 	default:
 		return ""
 	}
 }
 
+func HostScriptTimeoutMessage(seconds *int) string {
+	var timeout int
+	if seconds == nil {
+		timeout = int(scripts.MaxHostExecutionTime.Seconds())
```

Comment:
So nil means max timeout? I thought nil meant script succeeded. When will this method be called with nil input?

Reply:
in this case `nil` means no `script_execution_timeout` value exists in agent config, so it uses the default.